source/integrate-with-integration-environment/prove-users-identity.html.md.erb

```diff
@@ -7,15 +7,15 @@ review_in: 6 months
 
 # Prove your user's identity
 
-You must have authenticated your users before you can verify their identity.
+You must have authenticated your users before you can prove their identity.
 
-If you [requested identity assurance][integrate.choose-level-of-confidence], when you [retrieve user information with `/userinfo`](/integrate-with-integration-environment/authenticate-your-user/#retrieve-user-information), you’ll receive a response containing additional claims (user attributes). You may receive different claims, depending on how your user proved their identity.
+If you [requested identity proving][integrate.choose-level-of-confidence], when you [retrieve user information with `/userinfo`][integrate.retrieve-user-info], you’ll receive a response containing additional claims (user attributes). You may receive different claims, depending on how your user proved their identity.
```

Comment:
The term is "identity proofing", see https://datatracker.ietf.org/doc/html/rfc8485#section-2.1

But I'm not sure this change should be tied up with fixing the tables!

As an aside, it makes slightly more sense to ask for "identity assurance" because RPs don't ask us to necessarily prove someone's identity, they just ask that we _have_ (previously) proven it, and that the account is still secure (so the identity proof is protected with 2 factor auth etc). But "identity proofing" is fine too. "Proving" is just wrong (though frequently used in this programme - wrongly!).

Reply:
This is leftover from merging the changes that came in through the language consistency PR that was merged earlier in the week. 

We are aware that the programme has not been using consistent language, but 'proving' is used on GOV.UK to explain the concept (https://www.gov.uk/government/publications/identity-proofing-and-verification-of-an-individual/how-to-prove-and-verify-someones-identity). Hence we decided, that we will use 'proving' here, until we have got consensus on what language we want/should use. These discussions are being set up at the moment, and once that's concluded we will do a second iteration and update the wording.

Reply:
The word 'proving' isn't on that page at all? "Proving/prove" is the verb, but the noun for the process is "proofing". This is a grammar thing, I don't know why it is that way. But if you search "identity proving" on Google or something, you will mainly get non-relevant results for what we do, whereas "identity proofing" will get you information about what we do in GOV.UK One Login.

Reply:
The title talks about 'how to prove an identity', and since proving is the present participle of to prove, we are staying in line with how it is referred to there. Also, this is one of the recommended ways of talking about it in our UCD DI style guide.

We made this decision together with the technical architects of the Adoption pod, knowing that this will need re-discussing. I don't think, however, this is the right forum. I will raise an issue in the repo to pick this separately.

Reply:
Yep, as a verb like in that title it definitely makes sense. Agree this PR is not the place for it, better to separate it out.